Cargo.toml

```diff
@@ -21,10 +21,10 @@ rug = { version = "1.26", optional = true, default-features = false, features =
 glass_pumpkin = { version = "1", optional = true }
 
 [dev-dependencies]
-rand_core = { version = "0.9.2", default-features = false, features = ["os_rng"] }
+rand = { version = "0.10.0-rc.1", features = ["chacha"] }
+rand_core = { version = "0.10.0-rc.2", default-features = false }
```

Comment:
Since it doesn't require additional features for tests, this entry can be removed - there's already one in the main dependencies